### ISSUE TITLE
Citador does not work per 22 Dec 2019

### ISSUE THREAD

**Issue by tryoutuser** [nirewen/Citador#139]
Today (22 Dec 2019) i received "downloading updates 3 of 3" when launching discord and after that citador no longer working.

Please investigate and update.

**Comment by Arion-Kun** [comment]
Can confirm

**Comment by Ded111Let70rus** [comment]
when is the update?

**Comment by rikusen0335** [comment]
I'm confirming this too. It will be not worked on every update

**Comment by d3athrow** [comment]
probably the rich previews breaking it

**Comment by NightWolf1038** [comment]
[Here's](https://github.com/nirewen/Citador/files/4006128/Citador.zip) the update. It works but it wont clear the text area after sending the quote. I'm not at home to look for a fix.

**Comment by BrainRTP** [comment]
> [Here's](https://github.com/nirewen/Citador/files/4006128/Citador.zip) the update. It works but it wont clear the text area after sending the quote. I'm not at home to look for a fix.

Doesn't work.

**Comment by NightWolf1038** [comment]
> Doesn't work

Yeah, I know, I'll upload a fix when I have time

**Comment by DeepIndigo** [comment]
> 
> 
> > Doesn't work
> 
> Yeah, I know, I'll upload a fix when I have time

If it's any consolation, it worked for me until today.

**Comment by mateustebaldi** [comment]
Yeah, it's not working for several days. I'm clicking on the quote sign, but nothing happens.

**Comment by tryoutuser** [comment]
@NightWolf1038's fix was working until i believe three days ago(?), this tuesday, i received discord update again and his fix no longer working.

**Comment by HRudy94** [comment]
Also broken on my side... @NightWolf1038
It's been like a that for a few days... Hope to see your fix soon :)

**Comment by vperpl** [comment]
still not working ;/

**Comment by NightWolf1038** [comment]
[Update](https://github.com/NightWolf1038/Citador/releases/tag/1.10)

**Comment by tryoutuser** [comment]
Latest @NightWolf1038's update totally fixed it.
Dear @nirewen please accept the pull request.

**Comment by BlayZin** [comment]
I think discord fucked something up again

**Comment by DeepIndigo** [comment]
Can confirm. Citador functions completely vanished

**Comment by NightWolf1038** [comment]
I'm aware of it, it's gonna take some to fix it, so, please, be patient

**Comment by Lucasamiel0406** [comment]
I just lost 379 lines of nice css of my theme, Discord is amazing sometimes...
Also, don't rush NightWolf, he updated this days ago, he deserve some rest

**Comment by tryoutuser** [comment]
@BlayZin yes you're right.
I've just opened discord today and see new "new messages" indicator and bunch of quick reactions plus various shortcut. apparently discord made their own quote function. LUL

**Comment by jim2point0** [comment]
Every time this plugin is fixed, Discord updates and breaks it within days. If their built-in quote worked anywhere near as nice, I wouldn't mind so much. But this is just too good to not have.

**Comment by MasicoreLord** [comment]
Can't control Discord's update cycles.  If the native quote feature did work as well (w/ the automatically added timestamps and location included), this kind of plugin wouldn't be as desirable and likely to be discontinued (rich embed quotes, while they look nice, aren't really necessary).

**Comment by MasicoreLord** [comment]
Hope that explains things @jim2point0!

**Comment by tryoutuser** [comment]
@jim2point0 lmao exactly! so good to not have this plugin.

**Comment by AgreeToMyTerms** [comment]
I really hope it gets updated soon. The build-in Discord quote plugin just doesn't do it for me.

**Comment by ValtharionSS** [comment]
Hope someone's still working on updating this, this plugin was very convenient.

**Comment by lollilol** [comment]
updates on this? discords built in quoter is just cheap and often does not work (for example when you write the message and then clicking the quote button) (and its just not so pretty)

**Comment by tryoutuser** [comment]
@lollilol no news about this yet

**Comment by jim2point0** [comment]
If I knew even a little bit about modifying Discord plugins I'd give it a shot. But it's hard to tell from the source where it got broken. There's a lot in there and it's not exactly documented :(

**Comment by ldean14** [comment]
Is anyone still working on this? Would be nice to know if someone will get an update out eventually :/

**Comment by SasDaGreat** [comment]
Are there any plans for a future update? As of 27.01.21, the latest version of the plugin posted in this thread doesn't seem to work.

**Comment by tryoutuser** [comment]
@SasDaGreat unfortunately no more update i saw either
